modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -172,6 +172,14 @@ def build_claim_structure(data:, lighthouse_id:, upstream_id:) # rubocop:disable
           }
         end
 
+        def get_contention_list(data)
+          if data[:contentions].present? && data[:contentions].is_a?(Array)
+            data[:contentions].collect(&:strip)
+          elsif data[:contentions].present? && data[:contentions].is_a?(String)
+            [data[:contentions]]
+          end
+        end
```

Comment:
is it possible to get an array? my understanding is that we get back a string here that's comma separated. 

I believe we just need to add `.collect(&:strip)` at the end of the old :157.

Reply:
I ran across a "contention list" that was a String in the test data, and it caused a 500, so I added a catch for that.